iroha-lib/model/Tx.hpp

```diff
@@ -34,7 +34,8 @@ class Tx {
 
     Tx& addAssetQuantity(
             const std::string& account_id,
-            const std::string& role_name);
+            const std::string& role_name,
+            const std::string& title);
```

Comment:
BTW. I noticed that in Tx.cpp the name of the parameter `role_name` is `amount`. Could You please correct name into `role_name` -> `amount` in the method?